src/widgets/activity-group/model/factories/ListItemsFactory.ts

```diff
@@ -56,6 +56,8 @@ export class ListItemsFactory {
         progressRecord.pipelineActivityOrder + 1;
       item.activityFlowDetails.numberOfActivitiesInFlow =
         progressRecord.totalActivitiesInPipeline;
+      item.activityFlowDetails.activityFlowName =
+        progressRecord.entityName || activityFlow.name;
```

Comment:
In my opinion, it should always reference `progressRecord.entityName`.
Please create a migration that populates this field.

Reply:
Yes, simple migration is needed here. `activityFlow.name` is taken from react query cache - not appropriate for this case.